opentelemetry-exporter-gcp-trace/opentelemetry-exporter-gcp-trace.gemspec

```diff
@@ -14,39 +14,37 @@
 
 # frozen_string_literal: true
 
-require_relative "lib/opentelemetry/exporter/gcp/trace/version"
+require_relative "lib/opentelemetry/exporter/google_cloud_trace/version"
 
 Gem::Specification.new do |spec|
   spec.name = "opentelemetry-exporter-gcp-trace"
```

Comment:
If we're renaming the modules from `Gcp::Trace` to `GoogleCloudTrace`, should we consider renaming the gem to `opentelemetry-exporter-google_cloud_trace` accordingly?

Reply:
Left it to match the [python exporter](https://pypi.org/project/opentelemetry-exporter-gcp-trace/) ... may be should I leave the module name as such ?

Reply:
And renamed the module to match the other opentelemetry Ruby [exporters](https://github.com/open-telemetry/opentelemetry-ruby/blob/main/exporter/jaeger/opentelemetry-exporter-jaeger.gemspec)

Reply:
By convention, the gem name corresponds to the entrypoint module in the library. For example, in the Jaeger exporter, the name of the gem is "opentelemetry-exporter-jaeger", which corresponds to the main module "[OpenTelemetry::Module::Jaeger](https://github.com/open-telemetry/opentelemetry-ruby/blob/main/exporter/jaeger/lib/opentelemetry/exporter/jaeger.rb)". Ours should be similar. We should land on either:

* `opentelemetry-exporter-gcp-trace` with the module "OpenTelemetry::Exporter::GCP::Trace", or
* `opentelemetry-exporter-google_cloud_trace` with the module `OpenTelemetry::Exporter::GoogleCloudTrace`

Reply:
Sure @dazuma, makes sense. Preferring `google_cloud_trace` suffice as it aligns with other Ruby exporters and client library. will update the code.